docs/integrations/mirroring.md

```diff
@@ -79,13 +79,14 @@ def get_remote_data_command(client, args):
 ```
 
 ### get-modified-remote-data
-* GetModifiedRemoteDataArgs - this is an object created to maintain all the arguments you receive from the server in order to use this command.
+* **GetModifiedRemoteDataArgs** - this is an object created to maintain all the arguments you receive from the server in order to use this command.
 Arguments explanation:
-  - last_update - Date string representing the local time that the incident was last updated.
-* GetModifiedRemoteDataResponse - this is the object that maintains the format in which you should order the results from this function. You should use return_results on this object to make it work.
+  - *last_update* - Date string representing the local time that the incident was last updated.
+* **GetModifiedRemoteDataResponse** - this is the object that maintains the format in which you should order the results from this function. You should use return_results on this object to make it work.
 Arguments explanation:
-  - modified_incident_ids - a list of incidents that were modified since the last check, to later run the get-remote-data command on.
-  
+  - *modified_incident_ids* - a list of incidents that were modified since the last check, to later run the get-remote-data command on.
```

Comment:
I think we should describe both options here - you can return one entry of type JSON which is a string array with IDs or you can return multiple entries in which each entry is the full remote incident. In the second case, we will not even call `get-remote-data`. Also, in the second case, you must add the remote ID in the context of each entry under `mirrorRemoteId`.

Reply:
Also important to note that this method **does not** support files